src/App.tsx

```diff
@@ -10,11 +13,18 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Outlet />
         </div>
       </div>
+
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/home" element={<Navigate to="/" />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":slug" element={<PeoplePage />} />
+        </Route>
```

Comment:
The nested route for `PeoplePage` is defined, but it might be worth checking if the `PeoplePage` component is designed to handle both the base path `/people` and the nested `:slug` path. If `PeoplePage` is not intended to handle both, consider creating a separate component for the nested route.